runtime.md

```diff
@@ -23,6 +23,9 @@ This allows the hooks to perform cleanup and teardown logic after the runtime de
 * **`bundlePath`**: (string) is the absolute path to the container's bundle directory.
 This is provided so that consumers can find the container's configuration and root filesystem on the host.
 
+* **`annotations`**: (interface) holds runtime-specified information that is not structured by this specification.
```

Comment:
+1

On Wed, Dec 16, 2015 at 10:53 AM, Jonathan Boulle <notifications@github.com>
wrote:

> In runtime.md
> <https://github.com/opencontainers/specs/pull/188#discussion_r47816296>:
>
> > @@ -23,6 +23,9 @@ This allows the hooks to perform cleanup and teardown logic after the runtime de
> >  * **`bundlePath`**: (string) is the absolute path to the container's bundle directory.
> >  This is provided so that consumers can find the container's configuration and root filesystem on the host.
> >
> > +* **`annotations`**: (interface) holds runtime-specified information that is not structured by this specification.
>
> Eh, this is weird. We are already saying this is JSON encoded
> <https://github.com/opencontainers/specs/pull/188/files#diff-b84a8d65d8ed53f4794cd2db7e8ea731L11>,
> so we should just say the type here is an arbitrary JSON type (either an
> object or a value). "interface" is unhelpful.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/specs/pull/188/files#r47816296>.
>